build/index.html

```diff
@@ -937,6 +937,26 @@ <h2>Date of birth</h2>
    <br>
    <br>
 
+   <h2>File upload</h2>
+    <div class="govcy-container">
+        <form action="" class="govcy-form">
+            <fieldset class="govcy-fieldset">
```

Comment:
The input should not be in a `<fieldset>`.

Reply:
<fieldset> removed as per comment.